src/updateAttribute.js

```diff
@@ -50,7 +50,7 @@ function setAttribute(node, attrName, attrValue, oldAttrValue, isSvgAttribute) {
     // get patched event handler
     const patchedHandler = getPatchedEventHandler(node, attrName, attrValue);
     
-    const isCaptureEvent = attrName.endWith('Capture');
+    const isCaptureEvent = attrName.substr(-7) === "Capture";
```

Comment:
This will break for the special case. We can use the logic below here and pass isCaptureEvent as param to getEventName, just check if getEventName is getting called from anywhere else.

Reply:
why will this break ?